tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -227,8 +227,6 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
   // quantization related passes.
   pm.addNestedPass<FuncOp>(disc_ral::createDiscCustomCallRewriterPass());
   pm.addNestedPass<FuncOp>(disc_ral::createDiscConvertFakeQuantOpPass());
-  pm.addNestedPass<FuncOp>(
```

Comment:
Why delete this？

Reply:
It is redundant. Code in line 231-237 has covered it.